examples/06_datetime_encoder.py

```diff
@@ -0,0 +1,159 @@
+"""
+Handling datetime features with the DatetimeEncoder
+==========================================
+
+We illustrate here how to handle datetime features with the DatetimeEncoder. The DatetimeEncoder
+breaks down each datetime features into several numerical features, by extracting relevant information from the
+datetime features, such as the month, the day of the week, the hour of the day, etc. Used in
+the SuperVectorizer, which automatically detects the datetime features, the DatetimeEncoder allows
+to handle datetime features easily.
+"""
+
+import warnings
+warnings.filterwarnings("ignore")
+
+###############################################################################
+# Data Importing
+# --------------
+#
+# We first get the dataset.
+# We want to predict the NO2 air concentration in different cities, based on the date and the time of measurement.
+import pandas as pd
+data = pd.read_csv("https://raw.githubusercontent.com/pandas-dev/pandas/main/doc/data/air_quality_no2_long.csv")
+y = data["value"]
+X = data[["city", "date.utc"]]
+X
+
+###############################################################################
+# Creating encoders for categorical and datetime features
+# -------------------------
+from sklearn.preprocessing import OneHotEncoder
+from dirty_cat.datetime_encoder import DatetimeEncoder
+
+cat_encoder = OneHotEncoder(handle_unknown="ignore")
+datetime_encoder = DatetimeEncoder(add_day_of_the_week=True,  # The day of the week is probably relevant
+                                   extract_until="minute")  # We're probably not interested in seconds and below
+
+from sklearn.compose import make_column_transformer
+
+datetime_columns = ["date.utc"]
+categorical_columns = ["city"]
+
+encoder = make_column_transformer((cat_encoder, categorical_columns),
+                                  (datetime_encoder, datetime_columns),
+                                  remainder="drop")
+
+
+###############################################################################
+# Transforming the input data
+# ----------------------------
+# We can see that the encoder is working as expected: the date feature has
+# been replaced by features for the month, day, hour, and day of the week.
+# Note that the year and minute features have been removed by the encoder because they are constant.
+X_ = encoder.fit_transform(X)
+feature_names = encoder.get_feature_names_out()
+feature_names
+
+###############################################################################
+# Features importance
```

Comment:
I think that I would move the feature importance section below the prediction example. It feels more natural to understand feature importances in such contexte